src/database/migrations/0001_01_01_000000_create_users_table.php

```diff
@@ -13,37 +13,21 @@ public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('username')->unique();
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
+            $table->unsignedTinyInteger('status')->default(0)->comment('0=not verified 1=verified 3=deactivated');
```

Comment:
did you add new column in database using migration? if so, please create a separate migration file when adding new column if the database table is already existing!